testsuite/python/lb_couette_xy.py

```diff
@@ -0,0 +1,145 @@
+#
+# Copyright (C) 2021-2023 The ESPResSo project
```

Comment:
Year range should probably include 2025.

Reply:
I changed it.